python/lsst/fgcmcal/fgcmBuildStarsBase.py

```diff
@@ -96,6 +96,16 @@ class FgcmBuildStarsConfigBase(pexConfig.Config):
         keytype=str,
         itemtype=str,
         default={},
+        deprecated=("This field is no longer used, and has been deprecated by "
+                    "DM-28088.  It will be removed after v22.  Use "
+                    "physicalFilterMap instead.")
+    )
+    # The following will not be necessary after Gen2 retirement.
+    physicalFilterMap = pexConfig.DictField(
+        doc="Mapping from 'physicalFilter' to band.",
```

Comment:
I've added a comment, but the other instruments won't be added in gen2-era, so this will be retired because we can programmatically get `filterDefinitions.physical_to_band` in gen3.

Reply:
"it is set to" -> "obs packages should set it to"